ecs-cli/modules/cli/local/up_app.go

```diff
@@ -159,14 +188,29 @@ func decryptSecrets(containerSecrets []*secrets.ContainerSecret) (envVars map[st
 	return
 }
 
-// upComposeFile starts the containers in the Compose config with the environment variables defined in envVars.
-func upComposeFile(config *composeV3.Config, envVars map[string]string) {
+// upCompose starts the containers in the Compose files with the environment variables defined in envVars.
+func upCompose(envVars map[string]string, basePath string, overridePaths []string) {
+	// Gather environment variables
 	var envs []string
 	for env, val := range envVars {
 		envs = append(envs, fmt.Sprintf("%s=%s", env, val))
 	}
+	// Need to add $PATH because of --build, see https://stackoverflow.com/a/55371721/1201381
```

Comment:
Interesting, did this error actually occur for you?

Reply:
Yeah 😝 previously we didn't have `--build` part of `local up` but since we want the users to be able to use a Dockerfile to override one of their containers I ran into this problem